spec/azure/host_cluster.go

```diff
@@ -0,0 +1,6 @@
+package azure
+
+type HostCluster struct {
+	// CIDR is the CIDR of the host cluster Virtual Network.
```

Comment:
I'd like to add in the godoc we need that to open SSH port between guest and host clusters.